gui/src/app/util/timeAgoString.ts

```diff
@@ -0,0 +1,28 @@
+const timeAgoString = (timestampMsec?: number) => {
+  if (timestampMsec === undefined) return "";
+  const timestampSeconds = Math.floor(timestampMsec / 1000);
+  const now = Date.now();
+  const diff = now - timestampSeconds * 1000;
+  const diffSeconds = Math.floor(diff / 1000);
+  const diffMinutes = Math.floor(diffSeconds / 60);
+  const diffHours = Math.floor(diffMinutes / 60);
+  const diffDays = Math.floor(diffHours / 24);
+  const diffWeeks = Math.floor(diffDays / 7);
+  const diffMonths = Math.floor(diffWeeks / 4);
+  const diffYears = Math.floor(diffMonths / 12);
```

Comment:
Since months is unused, diffWeeks/52 is probably a more accurate way to calculate years

Reply:
Okay I removed months, and am now using 365 days / yr. Not worrying about leap year for this.